methods/midnight.json

```
@@ -0,0 +1,11 @@
+{
+  // These fields are required
+  "name": "midnight",
+  "status": "registered",
+  "specification": "https://github.com/IAMXID/did-midnight",
+  // These fields are optional
```

JSON doesn't support inline comments.
```suggestion
  "name": "midnight",
  "status": "registered",
  "specification": "https://github.com/IAMXID/did-midnight",
```